CalibCalorimetry/HcalPlugins/src/HcalHardcodeCalibrations.cc

```diff
@@ -38,10 +36,10 @@ namespace {
     int maxDepthHB=hcaltopology.maxDepthHB();
     int maxDepthHE=hcaltopology.maxDepthHE();
 
-#ifdef DebugLog
-    std::cout << std::endl << "HcalHardcodeCalibrations:   maxDepthHB, maxDepthHE = " 
-	      <<  maxDepthHB << ", " <<  maxDepthHE << std::endl;
-#endif
+  /*
```

Comment:
this isn't in #13232 and is also not really the way it should show up (commented out "cout"s)